.github/workflows/ci_cargo.yml

```diff
@@ -34,7 +34,7 @@ jobs:
           - std: true
             hal: "Yes (all features)"
         esp-idf:
-          - version: release/v5.1
+          - version: v5.1
             name: v5.1
```

Comment:
If Im not mistaken, we should be able to remove `name` property since they are now the same